### PR TITLE
Add diskless SBD test

### PR DESCRIPTION
**Pull request by juadk**
Add diskless SBD test in the cluster initialization part.

- Related ticket: N/A
- Needles: N/A
- Verification run: [Node01](http://1a102.qa.suse.de/tests/91) [Node02](http://1a102.qa.suse.de/tests/92) [Node03](http://1a102.qa.suse.de/tests/93)

### CODE REVIEW

File: tests/ha/ha_cluster_init.pm [hunk 1]
```diff
@@ -29,8 +29,18 @@ sub run {
 
     # If we failed to initialize the cluster, trying again but in debug mode
     # Note: the default timeout need to be increase because it can takes time to join the cluster
-    if (script_run "ha-cluster-init -y -s $sbd_device $unicast_opt", $join_timeout) {
-        assert_script_run "crm -dR cluster init -y -s $sbd_device $unicast_opt", $join_timeout;
+    # Initialize the cluster with diskless or shared storage SBD
+    if (get_var('USE_DISKLESS_SBD')) {
+	record_info('SBD info', 'Using diskless SBD');
```
Comment (alvarocarvajald) [inline tests/ha/ha_cluster_init.pm]: travis is failing here. I guess you used a TAB instead of 8 spaces.

File: tests/ha/ha_cluster_init.pm [hunk 2]
```diff
@@ -29,8 +29,18 @@ sub run {
...
+        if (script_run "ha-cluster-init -y -S $unicast_opt", $join_timeout) {
+            assert_script_run "crm -dR cluster init -y -S $unicast_opt", $join_timeout;
+        }
+    }
+    else {
+	record_info('SBD info', 'Using shared storage SBD');
+	if (script_run "ha-cluster-init -y -s $sbd_device $unicast_opt", $join_timeout) {
```
Comment (alvarocarvajald) [inline tests/ha/ha_cluster_init.pm]: travis is also failing in these 2 lines.

File: tests/ha/ha_cluster_init.pm [hunk 3]
```diff
@@ -47,8 +57,8 @@ sub run {
     # Execute csync2 to synchronise the configuration files
     exec_csync;
 
-    # State of SBD
-    assert_script_run "sbd -d $sbd_device list";
+    # State of SBD if shared storage SBD is used
+    assert_script_run "sbd -d $sbd_device list" if !(get_var('USE_DISKLESS_SBD'));
```
Comment (alvarocarvajald) [inline tests/ha/ha_cluster_init.pm]: You could write this also as `assert_script_run "sbd -d $sbd_device list" unless get_var('USE_DISKLESS_SBD');`, whichever you think it's easier to read.

File: tests/ha/ha_cluster_init.pm (hunk 1 above)
Comment (juadk) [reply tests/ha/ha_cluster_init.pm]: thanks, I've just installed the good .perltidyrc.

File: tests/ha/ha_cluster_init.pm (hunk 3 above)
Comment (juadk) [reply tests/ha/ha_cluster_init.pm]: `unless` is definitively easier to read.

**Pull request merged**